Dockerfile

```diff
@@ -3,26 +3,20 @@ MAINTAINER Tim Perry <pimterry@gmail.com>
 
 USER root
 
-RUN apt-get update && \
-    export DEBIAN_FRONTEND=noninteractive && \
-    # Set password to temp-password - reset to random password on startup
-    echo mysql-server mysql-server/root_password password temp-password | debconf-set-selections && \
-    echo mysql-server mysql-server/root_password_again password temp-password | debconf-set-selections && \   
-    # Instal MySQL for data, node as the JS engine for uglifier
-    apt-get install -y mysql-server nodejs
-    
-COPY . /opt/staytus
+ENV DEBIAN_FRONTEND="noninteractive"
 
-RUN cd /opt/staytus && \
-    bundle install --deployment --without development:test
-
-ENTRYPOINT /opt/staytus/docker-start.sh
+ADD docker-start.sh /docker-start.sh
+ADD . /opt/staytus/
 
-# Persists all DB state
-VOLUME /var/lib/mysql
+RUN chmod 755 docker-start.sh && \
+    apt-get update && \
+    # Instal node as the JS engine for uglifier
+    apt-get install -y nodejs && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
```

Comment:
The `apt-get clean` here isn't necessary, and *maybe* some or all of the `rm` too (but I'm not sure there).

Automatic `apt-get clean` is baked into the Docker base images already, and gets run automatically for you: https://github.com/docker/docker/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105. You can see this in `/etc/apt/apt.conf.d/docker-clean` in the current image.